pyramid_oereb/standard/sources/plr.py

```diff
@@ -38,8 +37,8 @@ def __init__(self, **kwargs):
             source (dict): The configuration dictionary of the public law restriction
             hooks (dict of str): The hook methods: get_symbol, get_symbol_ref. They have to be provided as
                 dotted string for further use with dotted name resolver of pyramid package.
-            law_status (dict of str): The multiple match configuration to provide more flexible use of the
-                federal specified classifiers 'inForce' and 'runningModifications'.
+            law_status (dict of str): The text of the law status. It must be from code "inKraft" or
```

Comment:
same as above: I do not get exactly the meaning of this description.
Do you want to say it has to be one out of the three "inKraft", "AenderungMitVorwirkung" or "AenderungOhneVorwirkung" or is the emphasis on something else?